src/AutoMerge/guidelines.jl

```diff
@@ -129,33 +129,45 @@ function meets_repo_url_requirement(pkg::String; registry_head::String)
     end
 end
 
-function meets_sequential_version_number(old_version::VersionNumber, new_version::VersionNumber)
-    if new_version > old_version
-        diff = difference(old_version, new_version)
-        @debug("Difference between versions: ", old_version, new_version, diff)
-        if diff == v"1.0.0"
-            return true, "", :major
-        elseif diff == v"0.1.0"
-            return true, "", :minor
-        elseif diff == v"0.0.1"
-            return true, "", :patch
-        else
-            return false, "Does not meet sequential version number guideline", :invalid
-        end
+function _invalid_sequential_version(reason::AbstractString)
+    return false, "Does not meet sequential version number guideline: $reason", :invalid
+end
+
+function _valid_change(old_version::VersionNumber, new_version::VersionNumber)
+    diff = difference(old_version, new_version)
+    @debug("Difference between versions: ", old_version, new_version, diff)
+    if diff == v"0.0.1"
+        return true, "", :patch
+    elseif diff == v"0.1.0"
+        return true, "", :minor
+    elseif diff == v"1.0.0"
+        return true, "", :major
     else
-        return false, "Does not meet sequential version number guideline", :invalid
+        return _invalid_sequential_version("increment is not one of: 0.0.1, 0.1.0, 1.0.0")
     end
 end
 
+function meets_sequential_version_number(existing::Vector{VersionNumber}, ver::VersionNumber)
+    always_assert(!isempty(existing))
+    issorted(existing) || (existing = sort(existing))
+    idx = searchsortedlast(existing, ver)
+    idx > 0 || return invalid("version $ver less than least existing version $(existing[1])")
```

Comment:
This “invalid” needs to be updated (sorry, not sure how to do backticks from my phone)

Reply:
Done (I think).

Are you iPhone or Android? If you are iPhone, click the button in the lower left hand corner that says ".?123". Then, press and hold over the apostrophe, and options for the backtick should pop up.

Reply:
Ooh, thanks! `now I can write code :)`